pyscript.core/README.md

```diff
@@ -47,3 +47,46 @@ npm run server
 ### Dev Build
 
 Beside spinning the _localhost_ server via `npm run server`, the `npm run dev` will watch changes in the `./esm` folder and it will build automatically non optimized artifacts out of the box.
+
+## Integration Tests
+
+To keep it simple, and due technical differences between what we had before and what we actually need for core (special headers, multiple interpreters, different bootstrap logic), core integration tests can be performed simply running:
+
+```sh
+npm run test:integration
+```
+
+The package's entry takes care of eventually bootstrapping a localhost, start in parallel all tests, and shut down the server after, if any was bootstrapped.
+
+The tool to test integration is still _playwright_ but move things a bit faster (from my side) tests are written in JS.
```

Comment:
Same here.... I'm not sure I understand "(from my side)". So, I'd ask.. do we need that?  I'd be +1 on removing